.github/workflows/build.yml

```diff
@@ -66,22 +66,25 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y --no-install-recommends \
-            ninja-build cmake g++ libgtest-dev libsdl2-dev zlib1g-dev libspdlog-dev
+            ninja-build cmake g++ libspdlog-dev
```

Comment:
you can probably also remove spdlog, the dependency has been dropped in #341

Reply:
yeah, i noticed that - am fine to do, just figured i'd keep this to a minimal change

Reply:
While we're editing the file, we can add that change ;)